libvirttest/libvirt.go

```diff
@@ -207,6 +207,26 @@ var testDomainsReply = []byte{
 	0x00, 0x00, 0x02,
 }
 
+var testDomainMemoryStatsReply = []byte{
+	0x00, 0x00, 0x00, 0x38, // length
+	0x20, 0x00, 0x80, 0x86, // program
+	0x00, 0x00, 0x00, 0x01, // version
+	0x00, 0x00, 0x00, 0x9f, // procedure
+	0x00, 0x00, 0x00, 0x01, // type
+	0x00, 0x00, 0x00, 0x07, // serial
```

Comment:
`0x00, 0x00, 0x00, 0x00`

sorry, not obvious but the serial is managed automatically.